spec/controllers/admin/rooms_controller_spec.rb

```diff
@@ -0,0 +1,171 @@
+require 'spec_helper'
+
+describe Admin::RoomsController do
+  let(:admin) { create(:user) }
```

Comment:
You imply here the user will  be an admin because it is the first user. But you can make it explicit, as we have a factory called admin, instead of user. It will make more sense when reading the test, and the test will not fail if we change the code with the first user.